src/vehicle/router.cpp

```diff
@@ -42,7 +42,7 @@ namespace CityFlow {
     }
 
     Drivable *Router::getNextDrivable(int i) const {
-        if (i < planned.size() && i >= 0) {
```

Comment:
What's the meaning when `i < 0`? If it makes no sense, please change `int i` to `size_t i`